zerver/webhooks/honeybadger/view.py

```diff
@@ -0,0 +1,139 @@
+# Webhooks for external integrations.
+from typing import Any, Dict, Tuple
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view, has_request_variables
+from zerver.lib.webhooks.common import check_send_webhook_message, UnexpectedWebhookEventType
+from zerver.lib.request import REQ
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+import re
+
+RATE_EXCEEDED_TEMPLATE = "{message}. For more info: {url}"
+ERROR_RESOLVED_TEMPLATE = "The error {id} was resolved by {actor}. For more info: {url}"
+ERROR_OCCURRED_TEMPLATE = "{klass}: {message} The error({id}) occurred at: {last_notice}. For more info: {" \
+                          "url} "
+COMMENT_TEMPLATE = "On error id: {id} {author} commented \n```quote\n{body}\n```\nFor more info: {url}"
+CHECK_IN_TEMPLATE = "{message}. {name} was expected to report at: {expected_at} and reported at: " \
+                    "{reported_at}. For more info: {url}"
+ASSIGNED_TEMPLATE = "{message}. Error with id: {id} has been assigned to {name}. For more info: {url}"
+UP_AND_DOWN_TEMPLATE = "{message} For more info: {url}"
+
+
+def get_topic(message: str) -> str:
+    t = re.match(r'([[\w\W]*])', message)
+    if not t:
+        raise AssertionError("Unrecognised payload format")
```

Comment:
It was discussed here: https://chat.zulip.org/#narrow/stream/127-integrations/topic/honeybadger/near/832058

Reply:
Yes, that's one way of doing it. I'm just wondering if it would make more sense to return a default topic name, and use the full message string as the message, in case we are not able to parse it.

Reply:
https://chat.zulip.org/#narrow/stream/91-code-review/topic/.2314209.20honeybadger.20integration/near/834705

Started an conversation here. Let's look out for suggestions.